questions.md

```diff
@@ -44,6 +44,7 @@ Please keep in mind that this document is in a continuous state of flux, with ne
 ```
 - What do you think are the main architectural challenges associated with developing for the iPad, compared to developing for the iPhone?
 - Do you have any experience developing for the Apple Watch or tvOS? If so, what's your opinion on the differences and similarities of the various development environments in the Apple development ecosystem?
+- How do you handle state in your app(s)?
```

Comment:
That's a *very* open-ended question. How do you know if the answer is a good one or not? What kind of answers are you expecting here?

Reply:
I don't, it's mostly to see how the person thinks about this stuff.

Reply:
> I don't there's just one answer

Either you mean "I don't. There's just one answer" or "I don't (know that) there's just one answer". I can't tell which.